provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -465,6 +465,10 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 					PrivateHZValidator(),
 				},
 			},
+			"wait": schema.BoolAttribute{
```

Comment:
Lets have more clear attribute name, something like: `wait_for_create_complete`

Reply:
Maybe ``wait_for_ready_status``? To match the status users see for their clusters
```
$ rosa list clusters
ID                                NAME             STATE  TOPOLOGY
<id>  <name>  ready  Classic
```

Reply:
I also thought about this name, but I think the purpose of this wait is to wait for any success or failure.
I don't have strong opinion about it, up to you @bardielle :-)

Reply:
I'll prefer `wait_for_create_complete`